cssApp.css

```diff
@@ -89,73 +103,30 @@ html, body {
 	color: #4e4e4e;
     font-size: 50px;
 }
+/*sun color #fdc400*/
```

Comment:
??

Reply:
This is just in case that I want to modify colors of weather icons in the future. This is nice color for sun icon.